.github/CODEOWNERS

```diff
@@ -579,7 +579,7 @@ x-pack/plugins/serverless @elastic/appex-sharedux
 x-pack/plugins/serverless_observability @elastic/appex-sharedux
 packages/serverless/project_switcher @elastic/appex-sharedux
 x-pack/plugins/serverless_search @elastic/appex-sharedux
-x-pack/plugins/serverless_security @elastic/appex-sharedux
+x-pack/plugins/serverless_security @elastic/security-solution
```

Comment:
@clintandrewhall, I think you've mentioned we'd like to stay notified of changes to serverless plugins for now. 

if that's right, @semd could you also keep us or at least add us with #CC# to stay notified?

Reply:
👇 
https://github.com/elastic/kibana/blob/4d1bbbe6c0dd06c3bad17ae2d3ef86aa13c88233/.github/CODEOWNERS#L792